src/q4wine-gui/iconsettings.cpp

```diff
@@ -457,22 +457,18 @@ void IconSettings::cmdGetIcon_Click(){
             args << "-x";
             args << "-t" << "14";
 
-            QString tmpDir="";
-            QStringList list1 = fileName.split("/");
-
-            tmpDir.append(QDir::homePath());
-            tmpDir.append("/.config/");
+            QString tmpDir = QDir::tempPath();
```

Comment:
looks like this can be replaced with https://doc.qt.io/qt-5/qtemporarydir.html

Reply:
done: https://github.com/brezerk/q4wine/commit/11446bcaa272ea319e651a45f8e398b04e619f9d